cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html

```diff
@@ -25,24 +25,31 @@
         {% import 'v1/includes/molecules/translation-links.html' as translation_links with context %}
         {{ translation_links.render(modifier_classes='block--flush-top') }}
 
-        {% for block in page.content %}
-            {% if block.block_type == 'notification' %}
-                {% set has_notification = true %}
-                {% include_block block %}
-            {% endif %}
-        {% endfor %}
-
-        <section class="ask-search block block--sub{% if has_notification == false %} block--flush-top{% endif %}">
-            {{ ask_search.render( language=page.language, is_subsection=False ) }}
-        </section>
-
         {% for block in page.content %}
             {% if block.block_type != 'notification' %}
                 {% include_block block %}
             {% endif %}
         {% endfor %}
     </div>
 
+    <div class="m-notification
```

Comment:
I wonder if we could add a new type (`help`?) to the notification molecule template so that could be used to output the notification content here.